packages/admin-panel/src/pages/resources/ResourcePage.js

```diff
@@ -55,13 +55,10 @@ export const ResourcePage = ({
           baseFilter={baseFilter}
           defaultFilters={defaultFilters}
           defaultSorting={defaultSorting}
+          ConfirmDeleteModalComponent={ConfirmDeleteModalComponent}
```

Comment:
Sorry what was this change?

Reply:
@edmofro I was a bit confused by this, as when I removed the {...editConfig} from the ResourcePage component, my eslint was not happy about this other prop not being declared and used. So would appreciate if you have a better idea?

Reply:
Quick call?

Reply:
@edmofro thanks. Should be all good with the latest commit now